apisix/constants.lua

```diff
@@ -33,6 +33,7 @@ return {
         ["/protos"] = true,
         ["/plugin_configs"] = true,
         ["/consumer_groups"] = true,
+        ["/kms/vault"] = true,
```

Comment:
`["/kms"] = true` is enough. We need to load all kms configs on init_worker, not just vault.

Reply:
Add both `/kms` and `/kms/vault` makes it work.
Because we need to get all resources both for `/kms` and `/kms/vault` prefix.